nautobot_ssot_infoblox/utils/client.py

```diff
@@ -29,6 +30,24 @@ def parse_url(address):
     return urllib.parse.urlparse(address)
 
 
+def get_default_ext_attrs(review_list: list) -> dict:
+    """Determine the default Extensibility Attributes for an object being processed.
+
+    Args:
+        review_list (list): The list of objects that need to be reviewed to gather default Extensibility Attributes.
+
+    Returns:
+        dict: Dictionary of default Extensibility Attributes for a VLAN View, VLANs, Prefixes, or IP Addresses.
+    """
+    default_ext_attrs = {}
+    for item in review_list:
+        pf_ext_attrs = get_ext_attr_dict(extattrs=item.get("extattrs", {}))
+        for k in pf_ext_attrs:
```

Comment:
nitiest of nits: Would `attr` be more appropriate than `k`? I see a single letter loop variable and my mind immediately goes to integers.